# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.16)
aclocal -I m4 -I gl/m4; autoheader; autoconf; automake -a -c
./configure --libexecdir=/lib/findutils --prefix=

# build
make

# install
install -d $DESTDIR/bin
install find/find $DESTDIR/bin
install locate/locate $DESTDIR/bin
install locate/updatedb $DESTDIR/bin
install xargs/xargs $DESTDIR/bin
install -d $DESTDIR/lib/findutils
install locate/frcode $DESTDIR/lib/findutils

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/findutils-build.pc << EOF
Name: findutils
Version: 4.9.0
Description: GNU utilities to find files
Requires: $REQUIRES

exec=\\
/bin/find \\
/bin/locate \\
/bin/updatedb \\
/bin/xargs \\
/lib/findutils/frcode
EOF

exit

# patch
--- findutils-4.9.0.orig/configure.ac
+++ findutils-4.9.0/configure.ac
@@ -44,6 +44,7 @@
 
 AC_CONFIG_AUX_DIR(build-aux)
 AM_INIT_AUTOMAKE([no-dist-gzip dist-xz color-tests])
+AM_MAINTAINER_MODE
 AM_SILENT_RULES([yes]) # make --enable-silent-rules the default.
 
 AC_CONFIG_SRCDIR([find/pred.c])

# source
https://ftp.gnu.org/gnu/findutils/findutils-4.9.0.tar.xz
